### PR TITLE
add&test distributed, add test example for MST

### PR DESCRIPTION
**Pull request by Suxiaocai**
1.  Add&test distributed mode
2. Add test example for MST
3. Change MST set name
4. Update README

### CODE REVIEW

File: graph_algorithms/README.algorithms [hunk 1]
```diff
@@ -63,23 +63,26 @@ https://docs.tigergraph.com/graph-algorithm-library
...
 label_prop                      Label Propagation Method for Community Detection
-louvain                         Louvain Modularity Method for Community Detection
+louvain                         Parallel Louvain Modularity Method with Refinement for Community Detection
 pageRank                        PageRank measurement of relative influence of each vertex
+pageRank_wt                     Weighted PageRank
+pageRank_pers                   Personalized PageRank
 shortest_ss_no_wt               Single-Source Shortest Paths without weight
 shortest_ss_pos_wt              Single-Source Shortest Paths with positive weight
 shortest_ss_any_wt              Single-Source Shortest Paths
+Minimal Spanning Tree (MST)
```
Comment (victor-gsl) [inline graph_algorithms/README.algorithms]: First column should be the query's name: mst
Second column is short descriptiobn

File: graph_algorithms/algorithms/examples/conn_comp2.gsql [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CREATE BATCH QUERY conn_comp2 () FOR GRAPH social {
```
Comment (victor-gsl) [inline graph_algorithms/algorithms/examples/conn_comp2.gsql]: What is conn_comp2?
It seems it is not new but it is not documented.

File: graph_algorithms/README.algorithms (hunk 1 above)
Comment (victor-gsl) [inline graph_algorithms/README.algorithms]: First column should be the query's name: mst
Second column is short descriptiobn

File: graph_algorithms/algorithms/install.sh [hunk 3]
```diff
@@ -256,10 +256,10 @@ END
 
         # 6. Ask for query mode.
        echo; echo "Please choose query mode:"
-       select mode in "Single Node Mode" "Batch Mode"; do
+       select mode in "Single Node Mode" "Distributed Mode"; do
```
Comment (victor-gsl) [inline graph_algorithms/algorithms/install.sh]: Also, elsewhere in install.sh (not in the recent change): "similarity" is misspelled twice.
"similary" --> "similarity"

File: graph_algorithms/algorithms/examples/conn_comp2.gsql (hunk 2 above)
Comment (Suxiaocai) [reply graph_algorithms/algorithms/examples/conn_comp2.gsql]: It is for test and it will be removed.

**Comment by Suxiaocai** [comment]
I made change to the code according to Victor's change request. Also added cycle_detection template.

**Pull request merged**